modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -0,0 +1,139 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-host-dual-network-interfaces-in-the-install-config-yaml-file_{context}"]
+= Optional: Configuring host network interfaces for dual port NIC
+
+[IMPORTANT]
+====
+Support for Day 1 operations associated with enabling NIC partitioning for SR-IOV devices is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Before installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState to support dual port NIC.
+
+.Prequisites
+
+* Configure a `PTR` DNS record with a valid hostname for each node with a static IP address.
+* Install the NMState CLI (`nmstate`).
+
+[NOTE]
+====
+Errors in the YAML syntax might result in a failure to apply the network configuration. Additionally, maintaining the validated YAML syntax is useful when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
```

Comment:
This is kind of odd to state, since this is true of any invalid YAML, not sure if it's important for some reason :shrug:

Reply:
I was asked to add this note and I take your point maybe thre is something specific with NMState and probably OK to remind customers about importance of valid YAML.